Cargo.toml

```diff
@@ -15,6 +15,10 @@ flamegraph = ["inferno"]
 protobuf = ["prost", "prost-derive", "prost-build"]
 cpp = ["symbolic-demangle/cpp"]
 
+pyroscope = ["reqwest", "tokio"]
+default-tls = ["reqwest/default-tls"]
```

Comment:
How about letting the user choose tls version by include `reqwest` explicitly? It's strange to choose a tls implementation for pprof-rs.

Reply:
Do you mean rewrite the name of `default-tls` to `reqwest-default-tls` (and `rustls-tls` to `reqwest-rustls-tls`)?